### PR TITLE
Added custom floor selector guide

### PR DESCRIPTION
**Pull request by timMikkelsen**

### CODE REVIEW

File: sdks-and-frameworks/android/custom-floor-selector.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# Custom Floor Selector
+
+## How to implement a custom floor selector
+
+To implement a custom floor selector we expect you to already have a View with a Map using MapControl. Start by creating a class, in this example named; `CustomFloorSelector`. Extend the class with `FrameLayout` and implement `MPFloorSelectorInterface` as well as it's methods.
```
Comment (matiasholst) [inline sdks-and-frameworks/android/custom-floor-selector.md:5]: Maybe link to the section on setting up mapcontrol?

File: sdks-and-frameworks/android/custom-floor-selector.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    }
+
+    override fun setSelectedFloorByZIndex(zIndex: Int) {
+        mSelectedFloor = floorMap[zIndex]
+        floorBtn?.text = floorMap[zIndex]?.displayName
+    }
+}
+```
+
+Now all that should be left is adding this view component to your layout containing the map and adding it to your `MPMapConfig` when creating `MapControl`. Here is an example layout where the floor selector is added to a layout.
```
Comment (matiasholst) [inline sdks-and-frameworks/android/custom-floor-selector.md:152]:
```suggestion
Now all that is left is adding this view component to your layout containing the map and adding it to your `MPMapConfig` when creating `MapControl`. Here is an example layout where the floor selector is added to a layout.
```

File: sdks-and-frameworks/android/custom-floor-selector.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+            mMap,
+            mMapView,
+            getString(R.string.mapbox_api_key),
+            true).setFloorSelector(floorSelector).build()
+    MapControl.create(mapConfig) { mapControl: MapControl?, miError: MIError? ->
+    }
+}
+```
+
+There we go, you should now have succesfully implemented a custom floor selector into your own app.
```
Comment (matiasholst) [inline sdks-and-frameworks/android/custom-floor-selector.md:187]:
```suggestion
There we go, you have now successfully implemented a custom floor selector into your own app.
```

**Review by matiasholst — CHANGES_REQUESTED** [review]
See 3 comments and suggestions.

**Comment by timMikkelsen** [comment]
> See 3 comments and suggestions.

@matiasholst Adressed the suggestions and comments now 👍

**Review by matiasholst — APPROVED** [review]

**Review by andreeaceachir142 — APPROVED** [review]
✅

**Pull request merged**